include/ddc/kernels/splines/splines_linear_problem_band.hpp

```diff
@@ -144,18 +144,18 @@ class SplinesLinearProblemBand : public SplinesLinearProblem<ExecSpace>
                 size(),
                 m_kl,
                 m_ku,
-                m_q.h_view.data(),
-                m_q.h_view.stride(
-                        0), // m_q.h_view.stride(0) if LAPACK_ROW_MAJOR, m_q.h_view.stride(1) if LAPACK_COL_MAJOR
-                m_ipiv.h_view.data());
+                m_q.view_host().data(),
+                m_q.view_host().stride(
+                        0), // m_q.view_host().stride(0) if LAPACK_ROW_MAJOR, m_q.view_host().stride(1) if LAPACK_COL_MAJOR
+                m_ipiv.view_host().data());
         if (info != 0) {
             throw std::runtime_error(
                     "LAPACKE_dgbtrf failed with error code " + std::to_string(info));
         }
 
         // Convert 1-based index to 0-based index
         for (int i = 0; i < size(); ++i) {
```

Comment:
```suggestion
        for (std::size_t i = 0; i < size(); ++i) {
```

It is nothing to do with this PR, but should this be `std::size_t`?

Reply:
Yes it would be better